llvm/include/llvm/ProfileData/InstrProf.h

```diff
@@ -357,6 +357,12 @@ void createPGONameMetadata(GlobalObject &GO, StringRef PGOName);
 /// the duplicated profile variables for Comdat functions.
 bool needsComdatForCounter(const GlobalObject &GV, const Module &M);
 
+/// \c NameStrings is a string composed of one of more possibly encoded
```

Comment:
```suggestion
/// \c NameStrings is a string composed of one or more possibly encoded
```

Do you mean `or` here?  (Yes, I understand that you've copied this from elsewhere.)